pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -2760,6 +2760,7 @@ dont-distribute-packages:
   - boolector
   - ccelerate-cuda
```

Comment:
Not related to this PR, but `ccelerate-cuda` is probably a mispelling of `accelerate-cuda`.

There is no `ccelerate-cuda` package on Hackage: https://hackage.haskell.org/package/ccelerate-cuda

Reply:
which begs the question. Why don‘t we have any evaluation errors caused by this?